CantoboardFramework/Keyboard/View/Status/StatusMenuHandler.swift

```diff
@@ -70,14 +69,16 @@ extension StatusMenuHandler where Self: UIView {
         guard statusMenu == nil else { return }
         FeedbackProvider.softImpact.impactOccurred()
         
-        var menuRows: [[KeyCap]] =  [
-            [ .changeSchema(.yale), .changeSchema(.jyutping) ],
-            [ .changeSchema(.cangjie), .changeSchema(.quick) ],
-            [ .changeSchema(.mandarin), .changeSchema(.stroke) ],
+        let menuRows: [[[KeyCap]]] = state.keyboardIdiom == .phone ? [
```

Comment:
Does this go off the screen on iPhones? Could you post a screen shot with the change on iPhone SE?